content/en/continuous_integration/quality_gates/_index.md

```diff
@@ -0,0 +1,173 @@
+This allows you to ensure that the code running in production is meeting high quality standards, reducing incidents and unwanted behaviors.
+
+## Set up Quality Gates
+
+There are two main steps required to set up and use Quality Gates:
+1. Create one or more rules in Datadog's UI.
+2. Invoke Quality Gates by using the `datadog-ci gate evaluate` command in your CI pipeline.
+
+## Create a rule
+
+To create Quality Gates rules for your organization, your user account must have the `quality_gate_rules_write` [permission][1].
+
+1. In Datadog, navigate to [**CI** > **Quality Gate Rules**][2] and click **+ New Rule**.
+2. Select the rule type. You can choose between `Static Analysis` and `Test`.
+3. Define the rule scope. The rule scope defines when the rule should be evaluated. For example, you can create a rule that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope. More information on [rule scopes](#rule-scope).
+4. Define the rule condition. The rule condition states in which scenario the rule fails, failing the related pipeline (if the rule is blocking). You can select one of the existing rule conditions for the rule type you have selected.
+
+   The following example shows how to create a static analysis rule that will fail when there are one or more static analysis violations with "error" status and "security" category being introduced in a specific commit:
+
+   {{< img src="ci/qg_rule_condition_sa_errors_security.png" alt="Rule for static analysis security errors" style="width:80%;">}}
+
+5. Select whether the rule should block the pipeline or not when it fails. Non-blocking rules are still evaluated, but
+they do not block the pipeline when they fail.
+6. Select the time window over which the query will be evaluated.
+7. Select a rule name that describes the rule that you are creating.
+8. Click **Save Rule**.
+
+
+## Invoking quality gates
+
+To use quality gates, your [`datadog-ci`][7] version should be higher or equal than `2.16.0`.
+
+You can invoke the Quality Gates evaluation by calling the [`datadog-ci gate evaluate`][4] command.
+
+This command:
+
+1. Retrieves all the rules that have [rules scopes](#rule-scope) and [custom scopes](#custom-scope) matching the current pipeline context (repository, branch, or custom scope passed in the command).
+2. Evaluates all the matching rules.
+3. If one or more blocking rules fail, the command fails as well, blocking the pipeline.
+
+<div class="alert alert-danger"><strong>Note:</strong> for the command to work properly, it's important that the events (tests, static analysis violations)
+are sent to Datadog <strong>before</strong> the <code>datadog-ci gate evaluate</code> command is executed.
+Otherwise, the rules might have an incorrect behavior due to the absence of the events.
```

Comment:
should we just say explicitly that we recommend adding the CLI command as the final job in your pipeline definition?

Reply:
Mmm not really, they can add it also in the middle of the pipeline. Actually, I think a future use case for Quality Gates is to stop further steps to be executed if the gate doesn't pass. Some sort of approvals based on rules